public/sass/components/_delete_button.scss

```diff
@@ -0,0 +1,49 @@
+.delete-button-container {
+  max-width: 24px;
+  width: 24px;
+  direction: rtl;
```

Comment:
Why is this needed?

Reply:
This is needed so that when you push the delete button it doesn't impact the rest of the row. So when the confirm is shown it is overflowing this div and that is why reverse direction is needed.

Reply:
Ah, ok. Could do with a comment then.